### PR TITLE
Deployment of the dke modules : py-nexusforge and py-bba-datafetch [BBPP82-632]

### PR DESCRIPTION
**Pull request by nabilalibou**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-sphinx-bluebrain-theme/package.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    homepage = "https://github.com/BlueBrain/sphinx-bluebrain-theme.git"
+    url = "https://pypi.io/packages/source/s/sphinx-bluebrain-theme/sphinx-bluebrain-theme-0.2.4.tar.gz"
+
+    version('0.2.4', sha256='4b3978dd3fe4aa3164ccb72e64117c4211432868e19cc9950fefdaa1d3b9bc1e')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-sphinx@2.0.0:', when='@0.4.1:', type=('build', 'run'))
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-sphinx-bluebrain-theme/package.py]: Uh… something about these versions does not match up. What Sphinx is required for `0.2.4`?

**Review by matz-e — COMMENTED** [review]
Got the following issue when building all this stuff:
```
[2021-09-20T17:04:11.016Z] ==> Warning: There is no checksum on file to fetch py-nest-asyncio@1.5.1 safely.

[2021-09-20T17:04:11.016Z] ==> Error: Will not fetch py-nest-asyncio@1.5.1

[2021-09-20T17:04:11.016Z] Add a checksum or use --no-checksum to skip this check.
```

File: var/spack/repos/builtin/packages/py-sphinx-bluebrain-theme/package.py (hunk 1 above)
Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-sphinx-bluebrain-theme/package.py]: Nice catch, probably forgotten after a copy/paste. I'll remove the  `when='@0.4.1:'`.

**Comment by nabilalibou** [comment]
> Got the following issue when building all this stuff:
> 
> ```
> [2021-09-20T17:04:11.016Z] ==> Warning: There is no checksum on file to fetch py-nest-asyncio@1.5.1 safely.
> 
> [2021-09-20T17:04:11.016Z] ==> Error: Will not fetch py-nest-asyncio@1.5.1
> 
> [2021-09-20T17:04:11.016Z] Add a checksum or use --no-checksum to skip this check.
> ```

I'll investigate on this matter. Is not having a checksum linked to a pypi package an issue that has been seen before ? I uninstalled/clean py-nest-asyncio@1.5.1 then reinstalled and push again to see if it will pass through the tests.

**Review by matz-e — APPROVED** [review]

**Pull request merged**